### PR TITLE
Stripped colour code symbols.

### PR DESCRIPTION
**Pull request by WeeSkilz**
Stripped colour code symbols through the use of RegEx, in response to #20

### CODE REVIEW

**Comment by totokaka** [comment]
I think this should be pulled.

I had a look at the regexp, in regards to how minecraft handles invalid formatting codes, and the behaviour with the current regexp should be correct.

**Comment by lukeroge** [comment]
I second the request to be able to keep color codes - I use this in my IRC bot and I translate them to IRC colors. I would also prefer the the stripped version be placed in a second variable, not the existing `description`, because it would cause a bunch of issues for handling people who have new/old versions of mcstatus installed!

**Comment by WeeSkilz** [comment]
@lukeroge I have made the original coloured description the same property and introduced a new one for stripped text, it's been added for a month

File: mcstatus/server.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ def status(self, retries=3, **kwargs):
                 pinger = ServerPinger(connection, host=self.host, port=self.port, **kwargs)
                 pinger.handshake()
                 result = pinger.read_status()
+                result.stripped_description = re.sub(r'\u00A7.', '', result.description)
```
Comment (pangeacake) [inline mcstatus/server.py]: Currently not compatible with python 2 because of unicode vs. string

**Comment by CoolCat467** [comment]
@connorlurring @ctian1 Isn't python 2 official depreciated now though?

File: mcstatus/server.py (hunk 1 above)
Comment (CoolCat467) [reply mcstatus/server.py:56]: Python 2 is officially depriciated now, so that's not a problem anymore

**Pull request closed without merging**